docs/workflow/testing/mono/testing.md

```diff
@@ -1,54 +1,83 @@
 # Running Tests using Mono Runtime
 
+Before running any tests, build mono in the way that you would like to test with.
+See the instructions for [Building Mono](../../building/mono/README.md)
+
 ## Running Runtime Tests
-We currently only support running tests against coreclr.  There are additional mono runtime tests in mono/mono, but they
-have not been moved over yet. Simply run the following command:
+* Build and run runtime tests against Mono JIT:
 
+    * Build and run one specific runtime test. From the root directory run the following command:
 ```
-dotnet build /t:RunCoreClrTests $(REPO_ROOT)/src/mono/mono.proj
+cd src/mono/netcore
+make run-tests-coreclr CoreClrTest="bash ../../artifacts/tests/coreclr/OSX.x64.Release/JIT/opt/InstructionCombining/DivToMul/DivToMul.sh"
 ```
 
-If you want to run individual tests, execute this command:
-
+    * Build and run all runtime tests. From the root directory run the following command:
```

Comment:
Similar feedback here to the comment for the individual tests. No bullet point, no root directory, etc.

Reply:
Fixed